controllers/search_controller.rb

```
@@ -13,7 +13,40 @@ class SearchController < ApplicationController
     get '/metadata' do
-      process_search
+      query = get_query(params)
+      options = get_ontology_metadata_search_options(params)
+      page, page_size = page_params
+
+      resp = search(Ontology, query, options)
+
+      result = {}
+      acronyms_ids = {}
+      resp.each do |doc|
+        resource_id = doc["resource_id"]
```

RuboCop (1.73)

[warning] Useless assignment to variable - `resource_id`.

(Lint/UselessAssignment)